coordinator/src/node.rs

```diff
@@ -60,20 +59,18 @@ impl Node {
             .filter(|channel| channel.is_usable && channel.counterparty.node_id == *trader)
             .collect::<Vec<_>>();
 
-        // should be exactly 1
+        if useable_channels.len() > 1 {
+            tracing::warn!(%trader, "Found more than one useable channel with trader");
```

Comment:
In the past we learnt that it was helpful to use consistent terminology in the logs so that searching through related logs is easier.

I would rename use `peer = %trader` to be consistent.

And we might want to start aligning i.e. check wherever we log a node ID and ensure that we log it as `peer` (or whatever name we like).